scripts/generate.sh

```diff
@@ -75,6 +84,10 @@ protoc_with_imports gogoslick encoding/x/protobuf/internal/wirepb/wire.proto
 protoc_with_imports gogoslick internal/examples/protobuf/examplepb/example.proto
 protoc_with_imports yarpc-go internal/examples/protobuf/examplepb/example.proto
 
+ragel -Z -G2 -o internal/interpolate/parse.go internal/interpolate/parse.rl
```

Comment:
Does ragel accept `-o -`? Is that perhaps the default behavior? Are you pondering what I’m pondering without a temp file?